scripts/common.sh.in

```diff
@@ -270,6 +274,11 @@ get_salt_container() {
 configure_salt_minion_local_mode() {
     local -r file_root="$BASE_DIR/salt"
 
+    # Workaround waiting for Rocky Linux support in Salt
+    if [[ $OS = "rocky" ]]; then
+        "$SALT_CALL" --local --retcode-passthrough grains.set os_family RedHat
+        "$SALT_CALL" --local --retcode-passthrough grains.set os CentOS
+    fi
```

Comment:
Can't we apply the grain state instead ?

Reply:
Well, not really since this state needs some pillar data, but maybe I can move this in another state.

Reply:
I've added an extra SLS for setting `os` and `os_family` grains and I know call it from there.